src/main/java/no/nav/folketrygdloven/beregningsgrunnlag/regelmodell/ytelse/frisinn/FrisinnGrunnlag.java

```diff
@@ -27,20 +29,36 @@ public FrisinnGrunnlag(List<FrisinnPeriode> frisinnPerioder, LocalDate skjæring
         return frisinnPerioder;
     }
 
-    public boolean søkerYtelseFrilans(LocalDate dato) {
-        return frisinnPerioder.stream().anyMatch(p -> p.inneholderDato(dato) && p.getSøkerYtelseFrilans());
+    public boolean søkerFrilansISøkandsperiode(LocalDate dato) {
+        return søkerStatusISøknadsperiode(dato, AktivitetStatus.FL);
     }
 
-    public boolean søkerYtelseFrilans() {
-        return frisinnPerioder.stream().anyMatch(FrisinnPeriode::getSøkerYtelseFrilans);
+    public boolean søkerNæringISøkandsperiode(LocalDate dato) {
```

Comment:
Er søkand ei spesiell type and? Er det nært beslekta den kjente og kjære stokkanda?